proto/spire/api/node/limits.go

```diff
@@ -4,8 +4,8 @@ const (
 	// Max burst values for ratelimiting
 	// Requests containing more than this number of
 	// operations will always be rejected
-	AttestLimit     int = 1
 	CSRLimit        int = 500
 	JSRLimit        int = 500
 	PushJWTKeyLimit int = 500
+	// AttestLimit is now configurable, its default value is set in the server run package.
```

Comment:
I'm not sure if we really need this comment. If we have it, I may reword it a bit removing the "now" word and point that is configured through the `attest_limit` server setting.

Reply:
I agree, it does not seems very useful. I'll remove it.